test/manual-scenarios/keyring/import account using private key.md

```diff
@@ -0,0 +1,46 @@
+# Manual test scenario for the 'Import account with private key' feature
+
+Below is a summary of the validations included:-
+
+* Validate that the account selector is present at the top of the wallet.
+* Validate that the 'Add account or hardware wallet' option is available at the bottom of the account selector list.
+* Validate that the 'Import account' option is available in the next menu.
+* Validate that the user is directed to the Import page upon selecting 'Import account'.
+* Validate that the user can paste their private key and click 'Import'.
+* Validate that the newly imported account appears in the account selector dropdown with an 'Imported' tag next to it.
+
+```markdown
+
+# Feature: Import Account with Private Key
+
+In order to manage multiple accounts
+As a user of the wallet extension
+I want to import an account using a private key
+
+# Scenario: Accessing the 'Add account or hardware wallet' option
+
+Given I am on the wallet interface
+When I click the account selector at the top of my wallet
+Then I should see a list of accounts
+And I should see the 'Add account or hardware wallet' option at the bottom of the list
+
+# Scenario: Selecting the 'Import account' option
+
+Given I have clicked the account selector at the top of my wallet
+And I see the 'Add account or hardware wallet' option at the bottom of the list
+When I select 'Add account or hardware wallet'
+Then I should be directed to the next menu
+And I should see the 'Import account' option
+
+# Scenario: Importing an account with a private key
+
+Given I am on the 'Import account' menu
+When I select 'Import account'
+Then I should be directed to the Import page
```

Comment:
Then I should be directed to the import dialog modal.

Reply:
@hjetpoluru  I have now modified the duplicates and the other mention, although I am not sure we are supposed to provide private keys. I will add and ask @chloeYue for an opinion as well?

Reply:
while i checked a bit in the official doc, we sometimes provide the private key example and sometimes not, so either way is **OK** for me. 
https://support.metamask.io/managing-my-wallet/secret-recovery-phrase-and-private-keys/how-to-export-an-accounts-private-key/
https://support.metamask.io/managing-my-wallet/accounts-and-addresses/how-to-import-an-account